src/otx/engine/engine.py

```diff
@@ -263,19 +280,28 @@ def train(
             logging.warning(msg)
             self.model.label_info = self.datamodule.label_info
 
-        self.trainer.fit(
-            model=self.model,
-            datamodule=self.datamodule,
-            **fit_kwargs,
-        )
+        with override_metric_callable(model=self.model, new_metric_callable=metric) as model:
+            self.trainer.fit(
+                model=model,
+                datamodule=self.datamodule,
+                **fit_kwargs,
+            )
         self.checkpoint = self.trainer.checkpoint_callback.best_model_path
+
+        if not isinstance(self.checkpoint, (Path, str)):
+            msg = "self.checkpoint should be Path or str at this time."
+            raise TypeError(msg)
+
+        best_checkpoint_symlink = Path(self.work_dir) / "best_checkpoint.ckpt"
+        best_checkpoint_symlink.symlink_to(self.checkpoint)
```

Comment:
This is a symlink so that there is no additional storage cost. To me, it seems better to have a fixed file path for the best checkpoint in the `work_dir`.

Reply:
Added in 81743a0